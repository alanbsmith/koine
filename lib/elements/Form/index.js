import styled from 'styled-components';

const Form = styled.form`
  box-sizing: border-box;
`;

export default Form;
