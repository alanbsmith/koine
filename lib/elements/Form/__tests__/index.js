import React from 'react';
import Form from '../index';
import renderer from 'react-test-renderer';


describe('Form Element', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Form />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
