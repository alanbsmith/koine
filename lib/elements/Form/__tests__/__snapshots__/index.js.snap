// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Element renders correctly 1`] = `
<form
  className="sc-bdVaJa jSHLiW"
/>
`;
